src/iam/signin.rs

```diff
@@ -154,22 +156,37 @@ pub async fn sc(
 									// The auth token was created successfully
 									Ok(tk) => Ok(Some(tk)),
 									// There was an error creating the token
-									_ => Err(Error::InvalidAuth),
+									_ => {
+										trace!("Error creating auth token");
+										Err(Error::InvalidAuth)
+									}
 								}
 							}
 							// No record was returned
-							_ => Err(Error::InvalidAuth),
+							_ => {
+								trace!("No record returned");
+								Err(Error::InvalidAuth)
+							}
 						},
 						// The signin query failed
-						_ => Err(Error::InvalidAuth),
+						Err(e) => {
+							trace!("Signin query failed {e:?}");
```

Comment:
This fails,
```
Signin query failed Tx("Cannot read or write data after any attempt to commit or roll back the transaction")
```
The error message doesn't seem to exist in either surrealdb or tikv projects.